README.md

```diff
@@ -1,42 +1,139 @@
-# my-package-name
+# openapi-client
+https://www.stitchdata.com/docs/developers/stitch-connect/api
 
-[![](https://img.shields.io/pypi/v/my-package-name.svg)](https://pypi.org/pypi/my-package-name/) [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+This Python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
-<!-- Short description of the package -->
+- API version: 0.1
+- Package version: 1.0.0
```

Comment:
Let's keep it at 0.1.0 for now. If we get to 100% API coverage then we'll go to 1.0.0